index.html

```diff
@@ -273,30 +273,42 @@ <h3>
             <tr>
               <th>Specification</th>
               <th>Description</th>
+              <th>Input Documents</th>
             </tr>
             <tr>
               <td>
-<a href="https://or13.github.io/lds-pgp2021/">PGP&nbsp;Cryptosuite</a>
+                  VC Data Integrity with PGP
```

Comment:
This is a bit confusing because it makes it seem like VC Data Integrity with PGP might not be a cryptosuite... I will try to think of alternate language.